common/src/main/resources/META-INF/openapi.json

```diff
@@ -720,6 +720,25 @@
                 "tags": [
                     "Admin"
                 ],
+                "parameters": [
+                    {
+                        "name": "X-Registry-Preserve-GlobalId",
+                        "description": "If this header is set to false, global ids of imported data will be ignored and replaced by next id in global id sequence. This allows to import any data even thought the global ids would cause a conflict.",
+                        "schema": {
+                            "type": "boolean"
+                        },
+                        "in": "header",
+                        "required": false
+                    },
+                    {
+                        "name": "X-Registry-Preserve-ContentId",
+                        "description": "If this header is set to false, content ids of imported data will be ignored and replaced by next id in content id sequence. The mapping between content and artifacts will be preserved. This allows to import any data even thought the content ids would cause a conflict.",
+                        "schema": {
+                            "type": "boolean"
+                        },
+                        "in": "header"
+                    }
+                ],
```

Comment:
Have these been added to the API spec in Apicurio Studio?  If yes, which one? :)